analyzer/consensus/consensus.go

```diff
@@ -611,6 +586,124 @@ func (m *Main) queueNodeEvents(batch *storage.QueryBatch, data *storage.Registry
 	return nil
 }
 
+func (m *Main) queueRegistryEvents(batch *storage.QueryBatch, data *storage.RegistryData) error {
+	eventInsertQuery := m.qf.ConsensusEventInsertQuery()
+	backend := analyzer.BackendRegistry
+
+	for _, event := range data.Events {
+		var ty analyzer.Event
+		var body []byte
+		var err error
+
+		switch {
+		case event.RuntimeEvent != nil:
+			ty = analyzer.EventRegistryRuntime
+			body, err = json.Marshal(event.RuntimeEvent)
+
+			if err != nil {
+				return err
+			}
```

Comment:
Here and in other newly added `switch/case` blocks: Can we handle `err != nil` after the `switch/case` block? It will increase code compactness+readability. We don't do any event-specific error handling, and I think it's unlikely that we will do so in the future.

Reply:
SG. Dunno if this is more kosher `Golang` but looks nicer to me as well!